hh-dep-monitoring-service/src/main/java/ru/hh/school/depmonitoring/dto/PageDto.java

```diff
@@ -0,0 +1,122 @@
+package ru.hh.school.depmonitoring.dto;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+
+public class PageDto<T extends Serializable> implements Serializable {
+    @NotNull
+    private T[] items;
+
+    private int found;
+
+    private int pages;
+
+    @NotNull
+    private Integer perPage;
+
+    private int page;
+
+    private PageDto() {
+        this(builder());
+    }
+
+    private PageDto(PageDtoBuilder<T> builder) {
+        this.items = builder.items;
+        this.found = builder.found;
+        this.pages = builder.pages;
+        this.perPage = builder.perPage;
+        this.page = builder.page;
+    }
+
+    public static <T extends Serializable> PageDtoBuilder<T> builder() {
+        return new PageDtoBuilder<>();
+    }
+
+    public T[] getItems() {
+        return items;
+    }
+
+    public Integer getFound() {
```

Comment:
```suggestion
    public int getFound() {
```
и в остальных геттерах